src/main/resources/templates/add_sighting.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html>
+    <head lang="en"
+    xmlns:layout="http://www.ultraq.net.nz/thymeleaf/layout"
+    xmlns:th="https://www.thymeleaf.org">
+    <link th:href="@{/css/addSighting.css}" rel="stylesheet" />
+    <link th:href="@{/css/application.css}" rel="stylesheet" />
```

Comment:
Should be indented as they are inside the `<head>`
```suggestion
        <link th:href="@{/css/addSighting.css}" rel="stylesheet" />
        <link th:href="@{/css/application.css}" rel="stylesheet" />
```